### ISSUE TITLE
DEBUG log entries from plugin contained in JUnitReporter output

### ISSUE THREAD

**Issue by Spatzier-Beck** [yoheimuta/protolint#81]
I have a protolint plugin with some custom defined rules. The protolint output should be written to a file. I use the following bash script to execute protolint (just a part of the script):
```bash
...
exec 2>$PROTOLINT_RESULT_FILE
protolint lint -plugin <custom plugin> -reporter junit .
...
```

After executing the script the $PROTOLINT_RESULT_FILE contains the following:
```xml
2019-10-07T09:35:25.066Z [DEBUG] plugin: starting plugin: path=/bin/sh args=[sh, -c, /usr/local/bin/kuka_rules]
2019-10-07T09:35:25.069Z [DEBUG] plugin: plugin started: path=/bin/sh pid=19
2019-10-07T09:35:25.069Z [DEBUG] plugin: waiting for RPC address: path=/bin/sh
2019-10-07T09:35:25.079Z [DEBUG] plugin.sh: plugin address: address=/tmp/plugin587097189 network=unix timestamp=2019-10-07T09:35:25.079Z
2019-10-07T09:35:25.081Z [DEBUG] plugin: using plugin: version=1
<?xml version="1.0" encoding="UTF-8"?>
  <testsuites>
      <testsuite tests="1" failures="0" time="0">
          <package>net.protolint</package>
          <testcase classname="net.protolint.ALL_RULES" name="All Rules" time="0"></testcase>
      </testsuite>
  </testsuites>
```

I saw that the [plugin mechanism](https://github.com/hashicorp/go-plugin) of protolint uses the [hclog logger](https://github.com/hashicorp/go-hclog). Its default output is stderr. The jUnitReporter of protolint also uses stderr. Which collides in my opinion.

How can I get rid of the go-plugin DEBUG log output in the protolint result file?

Used versions:
- protolint v0.17.0
- Go 1.12

Do you have any idea?

**Comment by yoheimuta** [comment]
@Spatzier-Beck Thank you for this report. 
I'm going to work on suppressing the plugin debug logs when -v is not specified.

**Comment by yoheimuta** [comment]
@Spatzier-Beck I have cut a release, including the fix.
Please confirm if it works for you.

- ref. https://github.com/yoheimuta/protolint/releases/tag/v0.17.1

**Comment by Spatzier-Beck** [comment]
@yoheimuta Thanks a lot for the very fast support :-) I tested it and it worked perfect.
